blocks/lead-art-block/features/leadart/default.jsx

```diff
@@ -68,6 +73,7 @@ export const LeadArtPresentation = (props) => {
 		/* istanbul ignore next */
 		const { default: AdFeature } = require("@wpmedia/ads-block/features/ads/default");
 		/* istanbul ignore next */
+		/* eslint-disable-next-line react/no-unstable-nested-components */
```

Comment:
I'm going to check if we already have a ticket for it, but this `AdBlock` functionality needs a refactor desperately.